### ISSUE TITLE
X11/20170314-GCCcore-6.3.0 requires harfbuzz? (osdep or module)

### ISSUE THREAD

**Issue by fgeorgatos** [hpcugent/easybuild-easyconfigs#4596]
fi.

```
[...]
checking for XRENDER... yes
checking for FREETYPE... configure: error: Package requirements (freetype2 >= 2.1.6) were not met:

Package 'harfbuzz', required by 'freetype2', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables FREETYPE_CFLAGS
and FREETYPE_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

**Comment by reedts** [comment]
We have the same problem on our cluster. Any fix yet?

**Comment by jlec** [comment]
`X11-20160819-foss-2016b` has the same problem. It is a CentOS 7 build.

**Comment by jlec** [comment]
I think harfbuzz needs to be added somewhere as a dependency. The freetype pkg-config file references harfbuzz. @boegel any suggestion?

**Comment by boegel** [comment]
This seems straightforward to fix, until you realise that freetype depends on HarfBuzz, and HarfBuzz depends on freetype, see also https://github.com/Homebrew/legacy-homebrew/issues/27847...

**Comment by boegel** [comment]
@jlec Another way to fix it is to remove the `harfbuzz` entry in the `freetype2.pc` file, see suggestion by @Helios07 at https://github.com/hpcugent/easybuild-easyconfigs/issues/3748#issuecomment-290347597

Maybe that's a better solution short-term than trying to work around the cyclic dependency we would introduce by including `Harfbuzz` as a dependency for `freetype`?

**Comment by boegel** [comment]
Apparently there's also a configuration option for `freetype` that could be used, i.e. `--with-harfbuzz=no`. When that is used, there's no mention of `harfbuzz` in `freetype2.pc`, so that's probably the best way out of this mess?

**Comment by jlec** [comment]
> @jlec Another way to fix it is to remove the harfbuzz entry in the freetype2.pc file, see suggestion by @Helios07 at #3748 (comment)

I doubt that is a good solution. It will be there for a reason.

> Apparently there's also a configuration option for freetype that could be used, i.e. --with-harfbuzz=no.

This is what we should do.

**Comment by boegel** [comment]
fixed in #4668
